### PR TITLE
fix: update env files

### PR DESCRIPTION
**Pull request by rephus**
Closes: #5291

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,38 +1,44 @@
...
-RUDDERSTACK_DATA_PLANE_URL=https://analytics.lightdash.com
-
-DBT_DEMO_DIR=/usr/app/examples/full-jaffle-shop-demo
+LIGHTDASH_LOG_LEVEL=debug
+LIGHTDASH_QUERY_MAX_LIMIT=5000
 
-SENTRY_DSN=''
 
-NODE_ENV=development
+NODE_ENV=production
```
Comment (IrakliJani) [inline .env:20]: do we need to change this?

**Review by IrakliJani — APPROVED** [review]
looks good to me!

we all have to rename `.env.local` to `.env.development.local` right?

**Comment by rephus** [comment]
> we all have to rename .env.local to .env.development.local right?

Only if you run docker and follow the contributing guide, 
if you don't use docker, this will not affect you 
if you use docker compose, and point to the right file with --env-file then you don't need to change anything

File: .env (hunk 1 above)
Comment (rephus) [reply .env:20]: Jose said this .env should represent production, so I think having NODE_ENV=production on .env makes sense to me

**Pull request merged**